backend/googlephotos/googlephotos.go

```diff
@@ -132,15 +132,26 @@ you want to read the media.`,
 			Default:  2000,
 			Help:     `Year limits the photos to be downloaded to those which are uploaded after the given year`,
 			Advanced: true,
+		}, {
+			Name:    "include_archived",
+			Default: false,
+			Help: `Also download archived media.
```

Comment:
This should probably say

`Also view and download archived media`

Reply:
Nice, pull request updated, thanks.